src/Gui/DlgAbout.cpp

```diff
@@ -534,6 +534,41 @@ void AboutDialog::linkActivated(const QUrl& link)
     licenseView->setSource(link);
 }
 
+void AboutDialog::addModuleInfo(QTextStream& str, const QString& modPath, bool firstMod)
+{
+    QFileInfo mod(modPath);
+    if (mod.isHidden()) {  // Ignore hidden directories
+        return;
+    }
+    if (firstMod) {
+        firstMod = false;
```

Comment:
Do you mean for `firstMod` to be passed by reference?

Reply:
Wtf? This is an old version of the patch... Hmm.. Will recheck later

Reply:
Wtf? This is an old version of the patch... Hmm.. Will recheck later

Reply:
Sorry, this was code from another PR #18530, which was rebased here by mistake